client/starwhale/api/_impl/evaluation.py

```diff
@@ -222,7 +222,7 @@ def _starwhale_internal_run_predict(self) -> None:
             ds.make_distributed_consumption(session_id=self.context.version)
             dataset_info = ds.info
             cnt = 0
-            idx_prefix = f"{_uri.name}-{_uri.version[:SHORT_VERSION_CNT]}"
+            idx_prefix = _uri.info().get("id") or _uri.name
```

Comment:
- raise if `uri.instance.is_cloud` and `uri.info().get("id")` is None?
- use `uri.project.name/uri.name` as prefix if dataset is local ds?

Reply:
done